PlayCover/AppInstaller/Installer.swift

```diff
@@ -183,7 +183,11 @@ class Installer {
 
     /// Generates a wrapper bundle for an iOS app that allows it to be launched from Finder and other macOS UIs
     static func wrap(_ baseApp: BaseApp) throws -> URL {
-        let location = PlayTools.playCoverContainer.appendingPathComponent(baseApp.url.lastPathComponent)
+        let info = AppInfo(contentsOf: baseApp.url
+            .appendingPathComponent("Info.plist"))
+        let location = PlayTools.playCoverContainer
+            .appendingPathComponent(info.displayName)
+            .appendingPathExtension("app")
```

Comment:
Does this address #304 ?

Reply:
Maybe, but it wasn't written to fix that bug

Reply:
We will find out then 👍